splunk_connect_for_snmp/inventory/loader.py

```diff
@@ -156,6 +105,64 @@ def configure_ui_database(mongo_client):
         profiles_ui_collection.drop()
 
 
+def add_groups_and_profiles_from_config_runtime(
+    groups_ui_collection, profiles_ui_collection
+):
+    groups = {}
+    all_profiles = {}
+    try:
+        with open(CONFIG_PATH, encoding="utf-8") as file:
+            config_runtime = yaml.safe_load(file)
+            if "groups" in config_runtime:
+                groups = config_runtime.get("groups", {})
```

Comment:
are we need this `config_runtime.get("groups", {})` if we already making `if "groups" in config_runtime`